src/parsers/index.js

```diff
@@ -74,8 +75,17 @@ const processStyledComponentsFile = (ast, absolutePath, options) => {
       extractedCSS.push(stylelintCommentsAdded)
       sourceMap = Object.assign(
         sourceMap,
-        getSourceMap(extractedCSS.join('\n'), wrappedContent, processedNode.loc.start.line)
+        getSourceMap(extractedCSS.join('\n'), wrappedContent, processedNode.quasi.loc.start.line)
       )
+      // Save interpolation lines
+      node.quasi.expressions.forEach((expression, i) => {
+        let l = node.quasi.quasis[i].loc.end.line + 1
```

Comment:
I was actually just about to comment that maybe we should not include the first line, but that's actually also what you're doing here right? The `+1` is excluding the line where the interpolation starts yeah? So if I for example did a single line interpolation `${someVariable}` I would still be checked for indentation errors yeah? If so all good and good on you for considering it as well

Reply:
Right, the indentation warning caused by the first line of the interpolations will not be filtered. Because indentations are wrote by users themselves, instead of generated by dummy comments padding.

Reply:
Yep, I agree that's exactly the behaviour we would want.

Reply:
I am thinking about renaming `interpolationLinesMap` as another name. How about `dummyCommentsMap` or something else?

Reply:
Hmm yeah, that's probably a fine idea to rename. So the purpose is to save where we have put in space filling comments in interpolation substitutions right... I feel like we should be able to come up with something better/even more specific than `dummyCommentsMap`, I guess it's also not really even a mapping right? More like a list. How about `dummyInterpolationLines`? Or `dummyInterpolationLinesList`?

This is definitely a subjective issue though, no real answer to this.